doc/supportedPlatformVersions.md

```diff
@@ -39,6 +39,7 @@
 - 7.0.100-preview.7.22377.5
 - 7.0.100-rc.1.22431.12
 - 7.0.100-rc.2.22477.23
+- 7.0.100-rc.2.22477.23
```

Comment:
(Please ignore) Adding a comment just so that we don't ignore these once we update the version to .NET 7 GA.